ConsumerEdition/DragonBoard-410c/Guides/dt-overlays.md

```diff
@@ -0,0 +1,100 @@
+# Instructions for Dynamically Loading Device Tree Overlays into Linux Kernel
+
+This document provides instructions for dynamically loading the device tree overlays (dtbo) into
+linux kernel running on DragonBoard410c.
+
+# Table of Contents
+
+- [1) Device Tree Compiler](#1-device-tree-compiler)
+    - [1.1) Installing the Compiler](#11-installing-the-compiler)
+- [2) Enable Overlay Support in Kernel](#2-enable-overlay-support-in-kernel)
+    - [2.1) Cloning and Building the kernel](#21-cloning-and-building-the-kernel)
+- [3) Load Overlays Dynamically](#3-add-overlays-dynamically)
+    - [3.1) Compiling the Overlays](#31-compiling-the-overlays)
+    - [3.2) Loading Overlays via Configfs](#32-inserting-overlays-via-configfs)
+     
+ ***
+ 
+ # 1) Device Tree Compiler
+ 
+ First of all we need to install device tree compiler (dtc) for compiling the source files (dts) into
+ overlays (dtbo). Overlay support was added to the mainline dtc by v1.4.2 only. But, the one which is available
+ as a package in debian based distros is v1.4.0. So, we need to install the compiler from source.
+ 
+ ## 1.2 Installing the Compiler
+ 
+```shell
+$ sudo apt-get install flex bison swig
+$ git clone git://git.kernel.org/pub/scm/utils/dtc/dtc.git
+$ cd dtc
+$ make
+$ sudo make install PREFIX=/usr
+```
+
+# 2) Enable Overlay Support in Kernel
+
+Next, overlay support needs to be enabled in the kernel. For which a custom kernel would be used other than
+the one available in Qualcomm landing page.
+
+## 2.1 Cloning and Building the kernel
+
+```shell
+$ export ARCH=arm64
+$ export CROSS_COMPILE=<path to your GCC cross compiler>/aarch64-linux-gnu-
```

Comment:
Nico,

Thanks a lot for your suggestion. I have updated the instruction and my tree.

Please take a look https://github.com/96boards/documentation/blob/master/ConsumerEdition/DragonBoard-410c/Guides/dt-overlays.md